c_api.h

```diff
@@ -14,6 +14,15 @@
 #define MAX_PURPOSE_LENGTH 64
 #define NETWORK_PROTOCOL_NAME_LENGTH 4
 
+#define AES_IV_SIZE 16
+#define SEQ_NUM_SIZE 8
+#define MAX_PAYLOAD_LENGTH 1024
+#define MAX_SECURE_COMM_MSG_LENGTH                                  \
+    1 + 2 + AES_IV_SIZE +                                           \
+        (((SEQ_NUM_SIZE + MAX_PAYLOAD_LENGTH) / AES_IV_SIZE) + 1) * \
+            AES_IV_SIZE +                                           \
+        MAC_KEY_SIZE
```

Comment:
I understand you want to make it a multiple of 16, but this code is very difficult to read. Can you define a helper macro for rounding up to the multiple of 16 and use it here, instead of putting all arithmetic here at once?

Reply:
```
#define ROUND_UP_TO_Y(X, Y) ((((X) / Y) + 1) * Y)
#define MAX_SECURE_COMM_MSG_LENGTH                                      \
    1 + 2 + AES_IV_SIZE +                                               \
        ROUND_UP_TO_Y(SEQ_NUM_SIZE + MAX_PAYLOAD_LENGTH, AES_IV_SIZE) + \
        MAC_KEY_SIZE  // Should be 1091
```
I did add a macro function, but still not sure if this is good enough to read. What do you think? @hokeun